vendor/assets/stylesheets/dvl/components/sidebar_nav.scss

```diff
@@ -0,0 +1,87 @@
+.sidebar_nav {
+  font-size: 0.9rem;
+  margin-bottom: 1rem;
+  > li {
+    @include font_smoothing;
```

Comment:
I know nothing about the font smoothing stuff, but is this something that we want to include on `html` or `body` instead of just this component?